src/ja/worker/proxy/remote.py

```diff
@@ -0,0 +1,14 @@
+"""
+LocalCentralServer is a single-class command line program.
+
+It queues the command on the worker socket.
+
+The commands are received on the standard input, and the worker response
+will be printed on the standard output.
+"""
+
+
+class LocalCentralServer:
```

Comment:
This should be called WorkerRemote and be a sub-class of the abstract Remote class in ja.common.proxy.remote